README.md

```diff
@@ -40,58 +40,36 @@ Please remember that this is the latest development build and is **not** tested
 
 ### For Android
 
-#### Quick and dirty
-
 Use the dockerized QField SDK.
 
 ```
-# get the latest tag from https://hub.docker.com/r/opengisch/qfield-sdk/tags/
-TAG=20191001
-cd QField
+QFIELD_SDK=$(awk -F "=" '/osgeo4a_version/{print $2}' sdk.conf)
+ANDROID_NDK_PLATFORM=android-29
 
-git submodule init
-git submodule update
+git submodule update --init --recursive
 
-docker run -v $(pwd):/usr/src/qfield docker.io/opengisch/qfield-sdk:$TAG /usr/src/qfield/scripts/docker-build.sh --rm
+docker run -e ANDROID_NDK_PLATFORM -v $(pwd):/usr/src/qfield docker.io/opengisch/qfield-sdk:$QFIELD_SDK /usr/src/qfield/scripts/docker-build.sh --rm
 ```
 
 This will put the apk into a subfolder `build-docker/out/build/outputs/apk`
 
-#### Go the hard way
-
-Build [OSGeo4A](https://github.com/opengisch/OSGeo4A)
-
-```sh
-cd QField
-
-git submodule init
-git submodule update
-
-cp config.pri.default config.pri
-# Edit config.pri
-```
-
-The advantage of this is, you will be able to build and deploy directly from QtCreator.
-
 ### For Desktop
 
 To build QField for a desktop environment:
 
-* Get QGIS development libraries.
+* Get QGIS development libraries. QField normally uses
+  bleeding edge QGIS code. Ideally install nightly builds
+  or compile it on your own and install to a custom prefix path.
 
```

Comment:
```suggestion
* install QField dependencies:

```bash
$ sudo apt install qtquickcontrols2-5-dev libqt5sensors5-dev qml-module-qtpositioning
```

```

Reply:
Can we separate that away since its platform specific?

Reply:
Sure! 

Since QField requires Qt 5.14, at least on Ubuntu LTS the dev needs to download Qt libs from the their website (https://www.qt.io/download) and all the requirements are already bundled there.

The list above is relevant when using the system's Qt version.

Worth mentioning this somehow in the text.

Reply:
That sounds like information that is temporary and needs maintenance?

Shouldn't we just document the system approach?

Reply:
What I was referring to before is that on my fedora system things are different.

Reply:
Now I see your point.
Can we at least describe the approach where qt.io download is used since everything is shipped there, is not platform specific and will rarely change?

Reply:
If we can do it this way, yes 👍